### PR TITLE
114364: Close decision cypress tests

### PR DESCRIPTION
**Pull request by FahadDarw**
**What is the change?**

Concerns Decisions User Story 8 - Close- Cypress Tests

**Why do we need the change?**
Per ticket 114364 Tests (automation and manual) - Close Decision

**What is the impact?**

minimal 

**Azure DevOps Ticket**
114364 & 102802

### CODE REVIEW

File: ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/decision/closeDecisionPage.ts [hunk 1]
```diff
@@ -0,0 +1,125 @@
+export class CloseDecisionPage
+{
+ 
+	public withSupportingNotes(supportingNotes: string): this {
```
Comment (mikestock-nimble) [inline ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/decision/closeDecisionPage.ts:4]: This looks like a duplicate of the method below

File: ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/decision/closeDecisionPage.ts [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+			message
+		);
+
+		return this;
+	}
+
+	
+  
+
+    public hasSupportingNotes(supportingNotes: string): this {
```
Comment (mikestock-nimble) [inline ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/decision/closeDecisionPage.ts:114]: Looks like a duplicate

**Review by mikestock-nimble — CHANGES_REQUESTED** [review]
Will discuss this offline, the close page seems to have the journey of closing a decision instead of just the operations available on the close page

File: ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/decision/closeDecisionPage.ts (hunk 1 above)
Comment (FahadDarw) [reply ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/decision/closeDecisionPage.ts:4]: Code refactored

File: ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/decision/closeDecisionPage.ts (hunk 2 above)
Comment (FahadDarw) [reply ConcernsCaseWork/ConcernsCaseWork.CypressTests/cypress/pages/caseActions/decision/closeDecisionPage.ts:114]: code refactored

**Review by mikestock-nimble — APPROVED** [review]

**Comment by FahadDarw** [comment]
> Will discuss this offline, the close page seems to have the journey of closing a decision instead of just the operations available on the close page

This has now been discussed and resolved, code had been refactored to reflect the above, Mike and I had worked together to resolve all the issue, automated test should be fully working according to the requirements.

**Pull request merged**